source/adapters/level_zero/command_buffer.cpp

```diff
@@ -1761,6 +2108,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferUpdateKernelLaunchExp(
   UR_CALL(validateCommandDesc(Command, CommandDesc));
 
   // We must synchronize mutable command list execution before mutating.
+  // FIXME Is this needed for the new path?
```

Comment:
I think we won't set `ZeActiveFence` in the new path since there won't be an output fence from the call